src/components/serviceRequest/PetInformationSection.tsx

```diff
@@ -81,7 +81,7 @@ export default function PetInformationSection(props: PetInformationSectionProps)
             <div className="grid col-12">
               <div className="col-fixed mr-3">{petInformationLabels.Species}</div>
               <div className="flex flex-wrap gap-3">
-                {speciesOptions.map((spec) => (
+                {speciesOptions ? speciesOptions.map((spec) => (
```

Comment:
I added seed data to the database so that species should always exist on startup. Even with that though, I still think this is a good change